src/components/Settings/Settings.js

```diff
@@ -1,23 +1,22 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
-import Title from 'components/Title';
 import Form from './Form';
 import ChangePasswordForm from './ChangePasswordForm';
 
-import styles from './Settings.module.scss';
+import { Container, Screen, SettingsTitle } from './Settings.styles';
```

Comment:
maybe `Container, Box`? instead of `Container, Screen`?

Reply:
I chose Screen because that's consistent with what was done in AuthWrapper. Lmk what you think

Reply:
I think that in the AuthWrapper it makes sense considering it's a "screen wrapper", but I don't know. @mollerjorge any input here?

Reply:
I usually use Container when what is beneath are a list of elements and I use Wrapper when what is beneath it's just one element. As mauri said I put screen because it works as a screen (viewport) wrapper but in this case as it's a component I would go with Content, Wrapper. I would use screen on a higher hirarchy component. PD: it's super opinionated this approach tho.